### ISSUE TITLE
Release json has breaking change for download URLs

### ISSUE THREAD

**Issue by bishal-pdMSFT** [dotnet/core#1977]
# Issue Title

[Release json](https://github.com/dotnet/core/blob/master/release-notes/releases.json) and csv file has used full dlc url for sdk version 2.1.403. This is a breaking behavior. Till now, each path was just a file name and full url could be generated by appending to base dlc/blob url. This is breaking Azure pipelines(formerly VSTS) .Net Core installer task.

# General
Is this change really required?
Are the blob urls not going to work?
Can this be made a non-breaking change? May be it can be done by adding a new property for full url instead of using existing properties.

**Comment by bishal-pdMSFT** [comment]
@leecow if You can I can contribute a PR to add a new property for full urls.

**Comment by NetTecture** [comment]
Well, I can say that up to NOW I still get that same error. Which means my whole pipeline for the whole team is down, becaue MS can not agree to write tasks that do not blow on a simple single minor update. When can we expec this to be fixed?

**Comment by leecow** [comment]
[chatted with Bishal offline and this captures what we discussed along with a little more detail on the 'why']

@bishal-pdMSFT this is the same type of issue as what broke you a few months ago. At that time, we talked about the need for you to deal with full URLs when present. We use this file to drive the .NET Core marketing site and logic is included in the parser. My recommendation is the same as the last time; this a change in robustness you really should adopt. 

We have moved to a different publishing pipeline which does not produce a stable release "container" (the guid portion of the url). Rather, every file has a unique path. I don't see a workable solution in this json format given the unique urls for every file, combined with the flat schema of the current json format, would make splitting the url and file difficult.

The json schema redesign is in final review now and it would be great to get your feedback so that it can be stable out of the gate. https://github.com/dotnet/core/pull/1960
